scripts/azureml-assets/azureml/assets/model/download_utils.py

```diff
@@ -94,7 +94,7 @@ def _onerror(func, path, exc_info):
         raise
 
 
-def run_cmd(cmd, cwd: Path | None = None, env: dict | None = None) -> int:
+def run_cmd(cmd, cwd: Path = None, env: dict = None) -> int:
```

Comment:
nit: i mean, if we want to keep the spirit of the original change, we can always import Optional from typing and do `Optional[Path] = None` and the like, right?